manifests/charts/istiod-remote/values.yaml

```diff
@@ -450,6 +450,8 @@ global:
   # The default value is "" and when caName="", the CA will be configured by other
   # mechanisms (e.g., environmental variable CA_PROVIDER).
   caName: ""
+  # whether to use autoscaling/v2 template for HPA settings
+  autoscalingv2API: true
```

Comment:
Do we need this in the istiod-remote chart values?

Reply:
Added comments. Not needed for remote, removed